### ISSUE TITLE
RSS feed per conference with new submissions

### ISSUE THREAD

**Issue by MacLemon** [pretalx/pretalx#440]
## Problem you are facing
As a pretalx user (and part of a conference organizing team) I'd like to integrate pretalx with existing communications infrastructures of the team. In our case, this is [Matrix](https://matrix.org/) which allows to integrate an RSS feed into the conference room that. It get's polled regularly by the Matrix server and new RSS-items are automatically posted into the channel.
This would excellently suit a notification mechanism for new submissions.

## Possible Solution
Generating an RSS feed per Conference that contains new talk submissions with a link to the submission page of the talk.

The RSS feed should offer a choice of requiring authentication or a simple “secret URL” type with a looong hash.
Team members could use a feed that requires a login, while bots and chat channels could use the secret URL type that doesn't leak a login to a third party provider and also spares the need to create a bot-team-member.

## Context
Having such an RSS feed available would positively motivate the team as they can see submissions coming in without having to read email where always some other emails are lurking as well.

Most likely RSS allows to integrate with other services as well. Such a feed could in fact carry a lot more items worth mentioning to the team. Like Addition of a team member, sending of email notifications has started/is done, approaching deadlines for closing CfP or opening reviews, etc.

In a later intallment this could provide even a personalized feed but bears the risk of creature feep and having pretax become a ticket system which should be avoided in favour of using an actual ticket system and having pretalx integrate with it. (Though RSS could mean a good way to actually do such an integration provided ticket systems can subscribe to such an automated feed.)

**Comment by rixx** [comment]
This is a very good proposal! For simplicity's sake, I'd prefer not to offer a choice of authentication options at first, and settle for one. Further auth options could be added as necessary. The simplest and easiest token would probably be the URL secret as proposed by you, with a button (labelled as dangerous, red, etc) to regenerate the URL.

**Comment by MacLemon** [comment]
Sounds splendid!

**Comment by saerdnaer** [comment]
Just a side note: For connection with Matrix and other real time chat software, Webhooks might be a better solution...

**Comment by rixx** [comment]
That is true, although for a good integration you probably want a plugin that uses that chat service's API (as the Slack plugin for pretalx does).
